search-app/src/cmr/search/site/data.clj

```diff
@@ -28,16 +31,52 @@
   (case tag
     "gov.nasa.eosdis" "EOSDIS"))
 
-(defn collection-data
+(def endpoint-get
```

Comment:
docs strings for most of this stuff would be helpful.

Reply:
+1 -- total oversight, thanks.

Sadly the tests are failing again (in anti-intuitive ways) so I haven't pushed the latest changes. Once they're running successfully again, I'll push so you can see the latest code, and then my last push will be all of the docstring updates.